From 5d329327574ced98bc76d48657e1ecd89594070f Mon Sep 17 00:00:00 2001
From: Fabricio Silva <hi@fabricio.dev>
Date: Sat, 8 Jul 2023 10:19:20 +0100
Subject: [PATCH] WebUI: Disable the network interfaces dropdown

Prevents the user to select the network interface in preferences.
This container will always bind "wg0" as to interface network.

---
 src/webui/www/private/views/preferences.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/webui/www/private/views/preferences.html b/src/webui/www/private/views/preferences.html
index d59734259..10fe3a4e7 100644
--- a/src/webui/www/private/views/preferences.html
+++ b/src/webui/www/private/views/preferences.html
@@ -903,7 +903,7 @@
                     <label for="networkInterface">QBT_TR(Network interface:)QBT_TR[CONTEXT=OptionsDialog]</label>
                 </td>
                 <td>
-                    <select id="networkInterface" style="width: 15em;">
+                    <select id="networkInterface" style="width: 15em;" disabled>
                     </select>
                 </td>
             </tr>
